sway-lsp/src/handlers/request.rs

```diff
@@ -83,8 +84,9 @@ pub async fn handle_goto_definition(
         .await
     {
         Ok((uri, session)) => {
+            let sync = state.sync_workspace.get().unwrap();
```

Comment:
nit: remove unwrap

Reply:
i see this is common for the arc; in this case ignore comment above

Reply:
[f8099e5](https://github.com/FuelLabs/sway/pull/7166/commits/f8099e56dc90bac4e9673117cae178c3bb9f8b50)